aw-datastore/src/datastore.rs

```diff
@@ -581,4 +598,49 @@ impl DatastoreInstance {
 
         return Ok(count);
     }
+
+    pub fn create_value(&self, conn: &Connection, key: &str, data: &str)
```

Comment:
Create isn't a proper name as it also can replace. Insert or put something similar is more suitable.

Reply:
Changed and fixed this in 6c7dd29